exec/cnex/lib/datetime.c

```diff
@@ -0,0 +1,57 @@
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
+#define timegm _mkgmtime
+#else
+#define _DEFAULT_SOURCE    // For timegm on linux
+#endif
+#include "datetime.h"
+
+#include <time.h>
+
+#include "array.h"
+#include "cell.h"
+#include "exec.h"
+#include "number.h"
+#include "stack.h"
+
+
+
+void datetime_gmtime(TExecutor *exec)
+{
+    Number t = top(exec->stack)->number;
+
+    time_t x = number_to_sint32(t);
+    struct tm *tm = gmtime(&x);
+
+    Cell *r = cell_createArrayCell(0);
+    Cell n; n.type = cNumber;
+    n.number = number_from_uint32(tm->tm_sec);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_min);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_hour); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_mday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_mon);  cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_year); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_wday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_yday); cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_isdst);cell_arrayAppendElement(r, n);
+    n.number = number_from_uint32(tm->tm_sec);  cell_arrayAppendElement(r, n);
```

Comment:
There seem to be one too many elements appended to this array here.

Reply:
Opps.  Fixed.